labgrid/remote/coordinator.py

```diff
@@ -960,6 +1043,25 @@ async def GetReservations(self, request: labgrid_coordinator_pb2.GetReservations
         reservations = [x.as_pb2() for x in self.reservations.values()]
         return labgrid_coordinator_pb2.GetReservationsResponse(reservations=reservations)
 
+    async def _release_session(self, client_session):
+        session = client_session.session
+
+        objs = self.session_manager.get_session_objs(session)
```

Comment:
I think it would be nicer if this returned a tuple. Then the `"places"` and `"reservations"` keys become an implementation detail of `SessionManager`:
```suggestion
        places, reservations = self.session_manager.get_session(id)
```

Reply:
I don't particularly think we need `_objs`, but I don't feel too strongly about it.